src/main/java/com/facebook/bitEncoderDecoder/Stage1.java

```diff
@@ -2,7 +2,14 @@
 
 public class Stage1 {
 
-    public String encode(String input){
-        return null;
+    public String encode(String input) {
+        StringBuilder encodeString = new StringBuilder();
+        for (int i = 0; i < input.length(); i++) {
+            encodeString.append(input.charAt(i));
```

Comment:
jest taka zasada DRY - don't repeat yourself. Dla niektórych to niemal zbrodnia jak masz takie same linie w kodzie :)